AK/URLParser.cpp

```diff
@@ -44,10 +46,184 @@ static Optional<DeprecatedString> parse_opaque_host(StringView input)
     return URL::percent_encode(input, URL::PercentEncodeSet::C0Control);
 }
 
-static Optional<DeprecatedString> parse_ipv4_address(StringView input)
+struct ParsedIPV4Number {
```

Comment:
A bit nit-picky but let's be consistent in the capitalization of the `V` - the spec uses `IPv4` so let's match that in struct names like this.

Reply:
thanks, appreciated - that does look better :^)